lisa/tools/servicectl.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any
+
+from lisa.executable import Tool
+
+
+class Service(Tool):
+    @property
+    def command(self) -> str:
+        return "service"
+
+    @property
+    def can_install(self) -> bool:
+        return False
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def _check_service_running(self, name: str) -> bool:
```

Comment:
It's better to keep consistent between systemctl and service. both called `is_running`?